app/great_battle.py

```diff
@@ -0,0 +1,16 @@
+from app.structuring_the_knight_personality import Knight
+
+
+class BattleOfTwoKnight:
+    @staticmethod
+    def battle_action_phase(knight_1: Knight, knight_2: Knight) -> None:
+
+        damage_for_first_knight = knight_2.power - knight_1.protection
+        damage_for_second_knight = knight_1.power - knight_2.protection
+        knight_1.hp -= max(0, damage_for_first_knight)
+        knight_2.hp -= max(0, damage_for_second_knight)
+
+    @staticmethod
+    def result_after_battle(knight: Knight) -> None:
```

Comment:
Method names should describe an action, do not use single-standing nouns or phrases as method names. You should utilize verbs here and in another method.

Reply:
👍